sdk/keyvault/keyvault/examples/singleSecret.ts

```diff
@@ -0,0 +1,42 @@
+import { SecretsClient } from "../src";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
```

Comment:
There are some things that I think we can take from: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples

1. The folder structure: Either `samples` or `examples`, but we should probably agree on one. Then, in service-bus we also have two folders `typescript` and `javascript`, which seems extremely valuable for users who might not be ready to work with TypeScript directly.
2. The README is pretty detailed about how to run these examples.
3. Each example has the Microsoft copyright at the top.
4. Each example has a detailed description of what they're intending to achieve.

Let me know what you think 🌞

Reply:
These are some great points for consistency. Can you file issues for each of the numbered points on the repo so that we can follow up on them?

Reply:
Issues created:

- [[JS] [KeyVault] Samples folder structure #3023](https://github.com/Azure/azure-sdk-for-js/issues/3023)
- [[JS] [KeyVault] Samples README #3024](https://github.com/Azure/azure-sdk-for-js/issues/3024)
- [[JS] [KeyVault] Samples CopyRights #3025](https://github.com/Azure/azure-sdk-for-js/issues/3025)
- [[JS] [KeyVault] Samples description in comments #3026
](https://github.com/Azure/azure-sdk-for-js/issues/3026)

Feel free to change them! \o/ Thank you!